portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -1040,7 +1041,16 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, cancer_study_
 	    return this.oql_query;
 	},
 	'getQueryGenes': function () {
-	    return OQL.genes(this.oql_query);
+		if (this.oql_query.trim().length > 0) {
```

Comment:
If you look at [line 179 from `global_variables.js`](https://github.com/thehyve/cbioportal/blob/geneset_frontend/portal/src/main/webapp/WEB-INF/jsp/global/global_variables.jsp#L179), `converted_oql` is generated by trimming another string. This `converted_oql` is what is passed to `initDatamanager` as `oql_query`. So it is not possible that `this.oql_query` is null:
```js
var converted_oql = oql_html_conversion_vessel.textContent.trim();
```